be/test/formats/json/nullable_column_test.cpp

```diff
@@ -142,4 +143,12 @@ TEST_F(AddNullableColumnTest, test_add_map) {
 
     ASSERT_EQ("[{'key1':'foo','key2':'bar','key3':'baz'}]", column->debug_string());
 }
+
+TEST_F(AddNullableColumnTest, test_invalid_json_convert) {
+    simdjson::ondemand::parser parser;
+    auto json = R"(  { "key" : "abc \ssn\"" }  )"_padded;
+    auto doc = parser.iterate(json);
+    simdjson::ondemand::value val = doc.find_field("key");
+    (void)convert_from_simdjson(val);
```

Comment:
it's a waste of the UT if bypass the return value validation. That's the whole point of this unit test. lol

Reply:
you can even change the `MAX_VALUE_LENGTH_FOR_ERRMSG`, to validate that the json string in error message DOES get truncated as expected.

Reply:
updated